extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.js

```diff
@@ -500,6 +498,17 @@ export class AmpStoryDevToolsTabPreview extends AMP.BaseElement {
       this.element
         .querySelector('.i-amphtml-story-dev-tools-device-chips')
         .appendChild(deviceSpecs.chip);
+    }).then(() => {
+      deviceSpecs.player
+        .getElement()
+        .addEventListener('storyNavigation', (event) => {
+          this.devices_.forEach((d) => {
+            if (d != deviceSpecs) {
+              d.player.show(null, event.detail.pageId);
+            }
+          });
+        });
+      deviceSpecs.player.load();
```

Comment:
@Enriqe is this what you mentioned during standup? If that's tripping us that'll definitely be annoying for external player users